routes/web.php

```diff
@@ -510,22 +510,38 @@
         });
     });
 
-    /* Routes related to the Quote Corner. */
-    Route::group(['prefix' => 'quotes', 'middleware' => ['member'], 'as' => 'quotes::'], function () {
-        Route::get('', ['as' => 'list', 'uses' => 'QuoteCornerController@overview']);
-        Route::post('add', ['as' => 'add', 'uses' => 'QuoteCornerController@add']);
-        Route::get('delete/{id}', ['as' => 'delete', 'middleware' => ['permission:board'], 'uses' => 'QuoteCornerController@destroy']);
-        Route::get('like/{id}', ['as' => 'like', 'uses' => 'QuoteCornerController@toggleLike']);
-        Route::get('search/{searchTerm?}', ['as' => 'search', 'uses' => 'QuoteCornerController@search']);
-    });
-
-    /* Routes related to the Good Idea Board. */
-    Route::group(['prefix' => 'goodideas', 'middleware' => ['member'], 'as' => 'goodideas::'], function () {
-        Route::get('', ['as' => 'index', 'uses' => 'GoodIdeaController@index']);
-        Route::post('add', ['as' => 'add', 'uses' => 'GoodIdeaController@add']);
-        Route::get('delete/{id}', ['as' => 'delete', 'uses' => 'GoodIdeaController@delete']);
-        Route::post('vote', ['as' => 'vote', 'uses' => 'GoodIdeaController@vote']);
-        Route::get('deleteall', ['as' => 'deleteall', 'middleware' => ['permission:board'], 'uses' => 'GoodIdeaController@deleteall']);
+    Route::get('quotes',['as'=>'quotes::list', function (Illuminate\Http\Request $request) {
+        return (new Proto\Http\Controllers\FeedBackController())->index($request, 'quotes');
+    }]);
+
+    Route::get('goodideas',['as'=>'goodideas::index', function (Illuminate\Http\Request $request) {
+        return (new Proto\Http\Controllers\FeedBackController())->index($request, 'goodideas');
+    }]);
```

Comment:
Feels like a 'beun oplossing', isn't there something nice for this?

Reply:
Yeah agreed, but it is the only fix that I can find that does not change the url and keeps the experience for our users therefore the same as it currrently is,